bitnami/dataplatform-bp1/README.md

```diff
@@ -143,12 +156,18 @@ Parameters below are set as per the recommended values, they can be overwritten
 | Parameter                   | Description                                      | Default                                                                                     |
 |:----------------------------|:-------------------------------------------------|:--------------------------------------------------------------------------------------------|
 | `spark.enabled`             | Switch to enable or disable the Spark helm chart | `true`                                                                                      |
+| `spark.master.webPort`      | Specify the port where the web interface will listen on the master                                                                  | `8080`   |
 | `spark.master.affinity`     | Spark master affinity for pod assignment         | Spark master pod Affinity rules set for best possible resiliency (evaluated as a template)  |
 | `spark.master.resources`    | CPU/Memory resource requests/limits for Master   | Spark master pods resource requests set for optimal resource usage                          |
-| `spark.worker.javaOptions`  | Set options for the JVM in the form `-Dx=y`      | No default                                                                                  |
 | `spark.worker.replicaCount` | Set the number of workers                        | `2`                                                                                         |
+| `spark.worker.webPort`      | Specify the port where the web interface will listen on the worker                                                                         | `8081`    |
+| `spark.worker.javaOptions`  | Set options for the JVM in the form `-Dx=y`      | No default                                                                                  |
 | `spark.worker.affinity`     | Spark worker affinity for pod assignment         | Spark worker pods Affinity rules set for best possible resiliency (evaluated as a template) |
 | `spark.worker.resources`    | CPU/Memory resource requests/limits for worker   | Spark worker pods resource requests set for optimal resource usage                          |
+| `spark.metrics.enabled`     | Start a side-car prometheus exporter                                                                                           | `false`       |
+| `spark.metrics.masterAnnotations`       | Annotations for enabling prometheus to access the metrics endpoint of the master nodes  | `{prometheus.io/scrape: "true", prometheus.io/path: "/metrics/", prometheus.io/port: "8080"}` |
+| `spark.metrics.workerAnnotations`       | Annotations for enabling prometheus to access the metrics endpoint of the worker nodes   | `{prometheus.io/scrape: "true", prometheus.io/path: "/metrics/", prometheus.io/port: "8081"}` |
+| `spark.metrics.resources.requests` | The requested resources for the metrics exporter container  | Spark exporter container resource requests for optimal resource usage size|
 
```

Comment:
The wavefront subcharts parameters should be added to this table

Reply:
done